src/src/CustomBuild/ESPEasyDefaults.h

```diff
@@ -320,13 +320,26 @@
 #endif
 
 #ifndef GITHUB_RELEASES_LINK_PREFIX
-# define GITHUB_RELEASES_LINK_PREFIX "<a href='https://github.com/letscontrolit/ESPEasy/releases' title='Click to show latest release on Github' target='_blank' style='font-size: 15px; text-decoration: none'>"
+# define GITHUB_RELEASES_LINK_PREFIX "<a href='https://github.com/letscontrolit/ESPEasy/releases' title='Click to show latest release on Github' target='_blank'>"
 #endif
 #ifndef GITHUB_RELEASES_LINK_SUFFIX
 # define GITHUB_RELEASES_LINK_SUFFIX "</a>"
 #endif
 
 
+
+/*
+TODO : 
+Following https://github.com/letscontrolit/ESPEasy/pull/3884/files#r774431094 in #3884
+The current "ESPEasy@mega-20211105" tag, should be replaced at build/release time with the correct tag while still allowing self-made builds
+*/
+#ifndef CDN_JS_CSS_URL
+#define CDN_JS_CSS_URL           "https://cdn.jsdelivr.net/gh/letscontrolit/ESPEasy@mega-20211105/static/"
```

Comment:
Please set this (after I resolve the merge conflicts) to the currently latest build: 20211224